app/connectors/ApplicationClient.scala

```diff
@@ -87,24 +85,11 @@ trait ApplicationClient {
     }
   }
 
-  def updateGeneralDetails(applicationId: UniqueIdentifier, userId: UniqueIdentifier, data: GeneralDetailsForm.Data, email: String)(
-    implicit
-    hc: HeaderCarrier
-  ) = {
-
+  def updateGeneralDetails(applicationId: UniqueIdentifier, userId: UniqueIdentifier, generalDetails: GeneralDetailsExchange)
```

Comment:
I think the GeneralDetailExchange here is called UpdateGeneralDetails in faststream backend. I think both should be named GeneralDetailExchange